### PR TITLE
Create settings page

### PR DESCRIPTION
**Pull request by SofieDunt**
## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/702429655/pulses/816231284)

What benefit does this bring to the end user? Or, what benefit does this bring to developers working in the codebase?

Settings page for users to edit their account settings or delete/deactivate their account, currently just styling

## This PR

Describe the changes required and any implementation choices you made to give context to reviewers.

Updated Settings.tsx and created settings.less to match the designs
- Used styled components for form titles and submit buttons to override internal styles
- Used Col grids to space out the form input fields

## Screenshots

Provide screenshots of any new components, styling changes, or pages.
Settings

## Verification Steps

What steps did you take to verify your changes work? These should be clear enough for someone to be able to clone the branch and follow the steps themselves.
- Visually checked that the page matches the designs
- Shrunk/maximized window size to check flexibility

### CODE REVIEW

File: src/containers/settings/Settings.tsx [hunk 1]
```diff
@@ -12,17 +38,116 @@ const Settings: React.FC = () => {
...
+                <Col span={formHalfItemSpan}>
+                  <Form.Item name="last-name">
+                    <Input placeholder="Last Name" />
+                  </Form.Item>
+                </Col>
+              </Row>
+              <Form.Item {...formLayout} name="username">
+                <Input placeholder="Username" />
+              </Form.Item>
+              <Form.Item {...formLayout} name="phone-number">
```
Comment (florisdobber) [inline src/containers/settings/Settings.tsx:73]: Sorry the designs aren't accurate in this but there's no phone number field.

File: src/containers/settings/Settings.tsx [hunk 2]
```diff
@@ -12,17 +38,116 @@ const Settings: React.FC = () => {
...
+                rules={[
+                  {
+                    required: true,
+                    message: 'Please input your current password!',
+                  },
+                ]}
+              >
+                <Input placeholder="Current Password" />
+              </Form.Item>
+              <Form.Item
```
Comment (florisdobber) [inline src/containers/settings/Settings.tsx:112]: Also not in the designs, but could you put text below the new password field saying "Your new password must be at least 8 characters". I think there's the option to put something below a field on a form field.

**Review by florisdobber — CHANGES_REQUESTED** [review]
Code looks great, just those two things which you couldn't have known about

File: src/containers/settings/Settings.tsx (hunk 2 above)
Comment (SofieDunt) [reply src/containers/settings/Settings.tsx:111]: I also added a rule to require having at least 8 characters for the new password input.
UpdatedSettings 
PasswordRule

Comment (florisdobber) [reply src/containers/settings/Settings.tsx:111]: Amazing!!

**Review by florisdobber — APPROVED** [review]
Looks good, thanks for making those changes!

File: src/containers/settings/settings.less [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.page-content-container {
```
Comment (jackblanc) [inline src/containers/settings/settings.less:1]: Is there any reason not to use a styled div for this? The same way we use other styled components. I think that'd be better for consistency.

File: src/containers/settings/Settings.tsx [hunk 4]
```diff
@@ -12,17 +40,118 @@ const Settings: React.FC = () => {
     <>
       <Helmet>
         <title>Settings</title>
-        <meta name="description" content="Description goes here." />
+        <meta
+          name="Settings"
+          content="The user's account settings page, where they can change their profile information and password or deactivate/delete their account."
+        />
       </Helmet>
```
Comment (jackblanc) [inline src/containers/settings/Settings.tsx:47]: This page is going to be protected, so we don't need Helmet for SEO

**Review by jackblanc — APPROVED** [review]
Looks great! Left some comments with room for improvement

File: src/containers/settings/settings.less (hunk 3 above)
Comment (SofieDunt) [reply src/containers/settings/settings.less]: No, I was just only using styled components when necessary to override internal styles. It's now updated to use a styled div.

**Pull request merged**